tests/cypress/start_cypress_tests.sh

```diff
@@ -60,6 +60,15 @@ else
   oc login --server=${CYPRESS_OPTIONS_HUB_CLUSTER_URL} --token=$CYPRESS_OPTIONS_HUB_TOKEN --insecure-skip-tls-verify
 fi
 
+if [[ $CYPRESS_OPTIONS_HUB_CLUSTER_URL =~ openshiftapps.com ]]; then
+  echo "ROSA cluster detected - excluding @rbac tests"
+  if [[ -z $CYPRESS_TAGS_EXCLUDE ]]; then
```

Comment:
To be safe, we should probably put these variables in quotes in the tests:
`"$CYPRESS_OPTIONS_HUB_CLUSTER_URL"`
`"$CYPRESS_TAGS_EXCLUDE"`

Reply:
Good idea